GoogleUtilities/Common/GULLoggerCodes.h

```diff
@@ -18,22 +18,38 @@
 
 typedef NS_ENUM(NSInteger, GULSwizzlerMessageCode) {
   // App Delegate Swizzling.
-  kGULSwizzlerMessageCodeAppDelegateSwizzling000 = 1000,                   // I-SWZ001000
-  kGULSwizzlerMessageCodeAppDelegateSwizzling001 = 1001,                   // I-SWZ001001
-  kGULSwizzlerMessageCodeAppDelegateSwizzling002 = 1002,                   // I-SWZ001002
-  kGULSwizzlerMessageCodeAppDelegateSwizzling003 = 1003,                   // I-SWZ001003
-  kGULSwizzlerMessageCodeAppDelegateSwizzling004 = 1004,                   // I-SWZ001004
-  kGULSwizzlerMessageCodeAppDelegateSwizzling005 = 1005,                   // I-SWZ001005
-  kGULSwizzlerMessageCodeAppDelegateSwizzling006 = 1006,                   // I-SWZ001006
-  kGULSwizzlerMessageCodeAppDelegateSwizzling007 = 1007,                   // I-SWZ001007
-  kGULSwizzlerMessageCodeAppDelegateSwizzling008 = 1008,                   // I-SWZ001008
-  kGULSwizzlerMessageCodeAppDelegateSwizzling009 = 1009,                   // I-SWZ001009
-  kGULSwizzlerMessageCodeAppDelegateSwizzling010 = 1010,                   // I-SWZ001010
-  kGULSwizzlerMessageCodeAppDelegateSwizzling011 = 1011,                   // I-SWZ001011
-  kGULSwizzlerMessageCodeAppDelegateSwizzling012 = 1012,                   // I-SWZ001012
-  kGULSwizzlerMessageCodeAppDelegateSwizzling013 = 1013,                   // I-SWZ001013
-  kGULSwizzlerMessageCodeAppDelegateSwizzlingInvalidAppDelegate = 1014,    // I-SWZ001014
-  kGULSwizzlerMessageCodeAppDelegateSwizzlingInvalidSceneDelegate = 1015,  // I-SWZ001015
+  kGULSwizzlerMessageCodeAppDelegateSwizzling000 = 1000,                 // I-SWZ001000
+  kGULSwizzlerMessageCodeAppDelegateSwizzling001 = 1001,                 // I-SWZ001001
+  kGULSwizzlerMessageCodeAppDelegateSwizzling002 = 1002,                 // I-SWZ001002
+  kGULSwizzlerMessageCodeAppDelegateSwizzling003 = 1003,                 // I-SWZ001003
+  kGULSwizzlerMessageCodeAppDelegateSwizzling004 = 1004,                 // I-SWZ001004
+  kGULSwizzlerMessageCodeAppDelegateSwizzling005 = 1005,                 // I-SWZ001005
+  kGULSwizzlerMessageCodeAppDelegateSwizzling006 = 1006,                 // I-SWZ001006
+  kGULSwizzlerMessageCodeAppDelegateSwizzling007 = 1007,                 // I-SWZ001007
+  kGULSwizzlerMessageCodeAppDelegateSwizzling008 = 1008,                 // I-SWZ001008
+  kGULSwizzlerMessageCodeAppDelegateSwizzling009 = 1009,                 // I-SWZ001009
+  kGULSwizzlerMessageCodeAppDelegateSwizzling010 = 1010,                 // I-SWZ001010
+  kGULSwizzlerMessageCodeAppDelegateSwizzling011 = 1011,                 // I-SWZ001011
+  kGULSwizzlerMessageCodeAppDelegateSwizzling012 = 1012,                 // I-SWZ001012
+  kGULSwizzlerMessageCodeAppDelegateSwizzling013 = 1013,                 // I-SWZ001013
+  kGULSwizzlerMessageCodeAppDelegateSwizzlingInvalidAppDelegate = 1014,  // I-SWZ001014
```

Comment:
we should keep the format of the message code enum consistently. Otherwise, it will be difficult to search for if the users provide the log with the message code. Here and below

Reply:
Agreed. While this change is just because of formatting. `kGULSwizzlerMessageCodeAppDelegateSwizzlingInvalidAppDelegate = 1014` was there for a while.